src/test/features/response/helpers/taskListBuilder.ts

```diff
@@ -34,5 +36,54 @@ describe('TaskListBuilder', () => {
         expect(taskList.tasks.map(task => task.name)).to.not.contain('Free mediation')
       })
     })
+
+    describe('Is Response Partially Rejected Due To task', () => {
+      beforeEach(() => {
+        stub = sinon.stub(ResponseDraft.prototype, 'isResponsePartiallyRejectedDueTo')
+      })
+
+      it('should be enabled when, when will you pay is available', () => {
```

Comment:
It makes more sense to say `should be enabled when claim is partially rejected due to amount being too high`. This way it is more explanatory.